File: Ginger/src/net/pic4pic/ginger/entities/AssetState.java
package net.pic4pic.ginger.entities;

public enum AssetState implements IntegerEnum {
    New(0),
    Disabled(1),
    Deleted(2);
    
    private final int value;
	
	private AssetState(int value) {
		this.value = value;
	}

	public int getIntValue() {
		return this.value;
	}
}

File: Pic4Pic/Pic4Pic.ObjectModel/Entities/AssetState.cs
using System;
using System.Runtime.Serialization;

namespace Pic4Pic.ObjectModel
{
    [DataContract()]
    public enum AssetState
    {
        [EnumMember]
        New = 0,

        [EnumMember]
        Disabled,

        [EnumMember]
        Deleted,
    }
}
